src/components/FileTree/index.spec.tsx

```diff
@@ -261,6 +262,7 @@ describe(__filename, () => {
       const treeFold = root.find(Treefold);
       expect(treeFold).toHaveProp('isNodeExpanded');
 
+      // eslint-disable-next-line @typescript-eslint/ban-types
       const isNodeExpanded = treeFold.prop('isNodeExpanded') as Function;
```

Comment:
As discussed above, can we not have a specific function type?

Reply:
Yep, we could and I added them.